lib/register_sources_bods/register/statement_loader.rb

```diff
@@ -55,11 +53,11 @@ def load_statements_children(statements)
         statements
       end
 
-      def load_statements_parents(statements)
+      def load_statements_parents(statements, max_levels:)
```

Comment:
Perhaps this should default to `MAX_LEVELS` to give same behaviour as before. But if all cases using it have been covered, then it's okay.